changelogs/unreleased/bump-libcurl-to-7.83.0.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
```

Comment:
Shouldn't we update configure options as we do for CMake in #7119?

Reply:
added